### PR TITLE
Feature/isaac quiz rubric

### PR DESCRIPTION
**Pull request by bh412**
Allows a rubric adding to isaacQuiz pages which is stored at the top level like the metadata. 

Will need a content change to existing isaacQuiz pages to make them editable in the editor, an empty rubric will be added

### CODE REVIEW

File: app/snippets/content_templates/isaacQuiz.json [hunk 1]
```diff
@@ -4,6 +4,17 @@
     "title": "New Quiz",
     "visibleToStudents": false,
     "published": false,
+    "rubric": {
+      "type": "content",
+      "encoding": "markdown",
+      "children": [
+        {
+          "type": "content",
+          "value": "_Enter rubric here or remove this section if not wanted_",
```
Comment (jsharkey13) [inline app/snippets/content_templates/isaacQuiz.json:13]: Is it actually possible to remove the section? If so, this wording is fine, and ignore me!

File: app/js/app/directives/react_classes/ContentBlock.js [hunk 2]
```diff
@@ -37,8 +47,21 @@ define(["react", "jquery"], function(React,$) {
...
+					</div>;
+
+					var quizSectionHeader = <div>
+						<h3>Quiz Sections</h3>
+					</div>
+				}
+
 				return (
 					<Block type="content" blockTypeTitle={this.props.blockTypeTitle} doc={this.props.doc} onChange={this.onDocChange}>
+						{rubric}
```
Comment (jsharkey13) [inline app/js/app/directives/react_classes/ContentBlock.js:63]: Is this safe when `rubric` is undefined on any other page type or content block? I assume you'd have noticed if it wasn't?

**Review by jsharkey13 — APPROVED** [review]

**Comment by jsharkey13** [comment]
I think this is fine. I don't _really_ understand the ins and outs, so I can only really be cursory in reviewing it :/

File: app/snippets/content_templates/isaacQuiz.json (hunk 1 above)
Comment (bh412) [reply app/snippets/content_templates/isaacQuiz.json:13]: You can press the cross in the visual editor to remove rubric and prevent it appearing on a quiz page, do you think some other wording would be better?

File: app/js/app/directives/react_classes/ContentBlock.js (hunk 2 above)
Comment (bh412) [reply app/js/app/directives/react_classes/ContentBlock.js:63]: Yeah, I'll have a double check when I look at the wording again but we do this elsewhere with 'conditional' sections, there's just a long section of bits in {} and they don't render if they're undefined

File: app/snippets/content_templates/isaacQuiz.json (hunk 1 above)
Comment (jsharkey13) [reply app/snippets/content_templates/isaacQuiz.json:13]: If the cross leave `rubric` null, or out of the JSON entirely, then nope this is great!

**Review by Morpheu5 — APPROVED** [review]

**Pull request merged**